metadata.yaml

```diff
@@ -9,12 +9,18 @@ summary: |
   MySQL is  a widely used, open-source relational database management system
   (RDBMS). MySQL InnoDB cluster provides a complete high availability solution
   for MySQL via Group Replication.
+source: https://github.com/canonical/mysql-operator
+issues: https://github.com/canonical/mysql-operator/issues
```

Comment:
For the history: https://discourse.charmhub.io/t/question-why-charm-mysql-ignores-issues-attribute-in-metadata-yaml/10029/3